src/main/java/io/github/thebusybiscuit/slimefun4/utils/tags/SlimefunTag.java

```diff
@@ -251,7 +261,21 @@ public enum SlimefunTag implements Tag<Material> {
     /**
      * All materials that are affected by gravity.
      */
-    GRAVITY_AFFECTED_BLOCKS;
+    GRAVITY_AFFECTED_BLOCKS,
+
+    /**
+     * Different types of Tools.
+     */
+    PICKAXES,
+    SHOVELS,
+    AXES,
+    HOES,
```

Comment:
aren't we missing special tools like compas,clock,shears,etc.
Funny to see these tags added btw today there was just a pr to Paper with these tags

Reply:
done